src/acl/Gadgets.cc

```diff
@@ -37,6 +38,12 @@ typedef std::set<ACL*> AclSet;
 /// Accumulates all ACLs to facilitate their clean deletion despite reuse.
 static AclSet *RegisteredAcls; // TODO: Remove when ACLs are refcounted
 
+// TODO: Upgrade ACL::name to SBuf and switch to std::unordered_map<SBuf,...>
+static auto AclNameLess = [](const char *a, const char *b) { return strcmp(a, b) < 0; };
```

Comment:
Old code in ACL::FindByName() uses strcasecmp(). Do we need it here instead of strcmp()?

Reply:
Good catch! Squid is not consistent about it, but we should go by ACL::FindByName(). Fixed in branch commit 0de1581.